frontend/tsconfig.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
Before this product enters production all the out commented lines should be removed. Thus they are in my mind fine to have for now, but should definetly be cleaned up as sooner rather than later.